### DISCUSSION TITLE
Announcing C and D levels

### DISCUSSION THREAD

**Discussion by demonkillerr** [tcet-opensource/.github#3] [Announcements]
### Greetings to everyone in the community!! 👋🏻

We have already revealed the leadership with regard to management as well as engineering to you, our members, on our discord event. Here is the official announcement regarding the **Office Bearers** (C level) and the **Directors of Engineering** (D Level), for the 2023-24 tenure.

If you would like to understand our structure, kindly visit announcement #2.

### Office Bearer:

- CEO - Himanshu Agarwal (@himanshu-03)
- CTO - Vishal Rawat (@vish-rt)
- COO - Mahima Churi (@Mahitej28)
- CFO - Asimvaibhav Tiwari (@asim-vaibhav-435)

### Directors of Engineering (L7)

- @tcet-opensource/frontend-team - Rahil Siddique (@rahilsiddique)
- @tcet-opensource/backend-team - Tejas Nair (@TejasNair9977) & Karthik Kaimaparambil (@kskarthik5)
- @tcet-opensource/devops-team - Akash Dubey (@Akash6222) & Vedant Pol (@VedantPol)
- @tcet-opensource/documentation-team - Nikita Shukla (@NikitaSShukla)
- @tcet-opensource/ui-ux-team - Kushal Singh (@kuxshl)
- @tcet-opensource/graphics-team - Khushi Gupta (@khushi-06)
- @tcet-opensource/ai-team - Prathik Shetty (@prathikshetty2002) & Hrishikesh Yadav (@Hrishikesh332)

### Executive Directors (L6)
- @tcet-opensource/frontend-team - Kamalika Ghora (@kamalika0363)
- @tcet-opensource/backend-team - Aastha Rai (@Aastha-S-Rai)
- @tcet-opensource/devops-team - Sarthak Gulhane (@beans9926)
- @tcet-opensource/documentation-team - Meet Jain (@MJ665)
- @tcet-opensource/ui-ux-team - Riya Gohil (@Riya-G23)
- @tcet-opensource/graphics-team - Aditya Rathod (@Rathod4408)
- @tcet-opensource/ai-team - Sanyam Jain (@sanyamjain0315)

**Comment by prathikshetty2002** [comment]
Congratulations 🎉 to all the C and D levels

**Comment by sanyamjain0315** [comment]
Stoked to be part of such a crucial organization ✨️

**Comment by Rhythm-shahane** [comment]
Really Appreciate to part of this Organisation and Congratulation 🎉 to all the C and D levels, they truly deserve it .
